apps/frontend/src/network/users/state.ts

```diff
@@ -35,17 +36,31 @@ const userState = selectorFamily<UserResponse | undefined, string>({
 
 export const useUserById = (id: string) => useRecoilValue(userState(id));
 export const usePatchUserById = (id: string) => {
+  const { getTokenSilently } = useAuth0();
   const authorization = useRecoilValue(authorizationState);
   const setPatchedUser = useSetRecoilState(patchedUserState(id));
   return async (patch: UserPatchRequest) => {
     setPatchedUser(await patchUser(id, patch, authorization));
```

Comment:
I would think that once the `setPatchedUser` completes, the profile should be updated, no matter what happens with the token, but on the environment it seems that the profile also waits for the token request to finish, and if that errors no update happens?